### PR TITLE
MUXFinalRelease packages should not have -prerelease tag

### PR DESCRIPTION
**Pull request by jevansaks**
Another bit of logic that didn't get ported over to the yaml pipeline.

### CODE REVIEW

File: build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ parameters:
   primaryBuildArch: x86
   buildFlavor: Release
   signConfig: ''
+  releasePackage: 'false'
```
Comment (licanhua) [inline build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: maybe officalRelease or just usePrereleaseTag are better than this name.
Another option is export prereleaseTag directly, and default is "prerelease".

Comment (jevansaks) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: Fair enough, i'll change it to "usePrereleaseTag".

Comment (jevansaks) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: Actually one of the problems I had was manipulating the queue-time variable at this level. The flag needs to be the opposite -- how about noPrereleaseTag?

Comment (licanhua) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: good

Comment (jevansaks) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: or maybe just add useReleasePackageTag?

Comment (licanhua) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: useReleasePackageTag and with the default setting opposite

**Pull request merged**